src/index.html

```diff
@@ -2,16 +2,122 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport" content="width=device-width, user-scalable=no, initiaPl-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <!-- PERSONAL INFORMATION -->
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <label class="form-field">Surname:
+          <input
+            type="text"
+            name="surname"
+            autocomplete="off"
+            required
+          >
+        </label>
+        <label class="form-field">Name:
+          <input
+            type="text"
+            name="name"
+            autocomplete="off"
+            required
+          >
+        </label>
+        <label class="form-field">How old are you?
+          <input
+            type="number"
+            name="age"
+            min="1"
+            max="100"
+            value="12"
+          >
+        </label>
+        <label class="form-field">Full date of birth:
+          <input type="date" name="dateOfBirth">
+        </label>
+        <label class="form-field">I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="polyciAgreement"
+            required
+          >
+        </label>
+      </fieldset>
+      <!-- RESISTRATION -->
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+        <label class="form-field">E-mail:
+          <input
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+            required
+          >
+        </label>
+        <label class="form-field">Password:
+          <input
+            type="password"
+            name="password"
+            minlength="4"
+            maxlength="20"
+            required
+          >
+        </label>
+      </fieldset>
+      <!-- INTERESTING FACTS ABOUT YOU -->
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you</legend>
+        <label class="form-field">Do you love cats?
```

Comment:
Don't use this common label, because when you click on the question, yes becomes selected